app/presenters/activity_presenter.rb

```diff
@@ -68,6 +68,16 @@ def recipient_country
     I18n.t("activity.recipient_country.#{super}")
   end
 
+  def requires_additional_benefitting_countries
+    return if super.nil?
+    I18n.t("activity.requires_additional_benefitting_countries.#{super}")
+  end
+
+  def intended_beneficiaries
+    return if super.blank?
```

Comment:
Do we need to return `"Not application"` here if the beneficiaries list is empty, or if `requires_additional_benefitting_countries` is `false`?

Reply:
I hadn't noticed this, thanks. The partial for activity details view has some logic in it so it won't display the field `intended beneficiaries` in the situations you've mentioned

Reply:
The only situation I can think of is in the case of an incomplete activity where `intended beneficiaries` haven't been set yet. In that case the field will be display on the activity details page but the value will be blank. I think if we display "Not applicable" could be confusing as the user wouldn't know if it's because the activity is incomplete or because the activity does not require any additional benefitting countries